### ISSUE TITLE
addListener is not triggered for "bounce / overscroll" values

### ISSUE THREAD

**Issue by IljaDaderko** [idiotWu/smooth-scrollbar#300]
## Environment
- Browser: Chrome Version 88.0.4324.182
- Version of smooth-scrollbar: 8.6.0

## Issue Summary

I am using `addListener` to send offset values to canvas. It works great, but it looks like it doesn't send any event when scrollbar is "bouncing".

## Current Behavior

Please see attached video, `addListener` doesn't send any events for "bounce" values.

## Expected Behavior

`addListener` should fire for "bounce" values

## Steps to Reproduce

```ts
import Scrollbar from 'smooth-scrollbar';
import OverscrollPlugin from 'smooth-scrollbar/plugins/overscroll';

Scrollbar.use(OverscrollPlugin);

const scrollbar = Scrollbar.init(myDivElement, {
      plugins: {
        overscroll: {
          effect: 'bounce'
        }
      }
});

scrollbar.addListener(data => console.log(data.offset.y));
```

## Demo

https://user-images.githubusercontent.com/3154053/109169531-9096ed80-7788-11eb-88c0-0c7fff61e7fc.mov

**Comment by idiotWu** [comment]
The overscroll plugin has its own [onScroll](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/overscroll.md#optionsonscroll) option:

```js
{
  plugins: {
    overscroll: {
      onScroll(position) {
        console.log(posision); // > { x: 12, y: 34 }
      }
    }
  }
}
```

**Comment by IljaDaderko** [comment]
I see, it only triggers on over-scroll / bounce values, so combination of both `overscroll.onScroll` and `addListener` is needed to support full scrolling. It's a bit unintuitive, perhaps `overscroll` should expose method for full scrolling events?

**Comment by idiotWu** [comment]
Yes, it's a little bit difficult to use because of some historical issues. The overscroll plugin was designed as an independent plugin for smooth scrollbar and the `onScroll` option was not available until issue #111. 

The following code calculates the real (?) scrolling offsets of the scrollbar:

```js
onScroll({x, y}) {
  x += this.scrollbar.scrollLeft;
  y += this.scrollbar.scrollTop;
}
```

**Comment by IljaDaderko** [comment]
Yeh that does the trick
